aws/test/__snapshots__/cloudwatch_collector.test.js.snap

```diff
@@ -23,8 +26,11 @@ Object {
   "ingest_timestamp": 1538769915817,
   "logEvents": Array [
     Object {
+      "Region": "region",
+      "accountId": "accountId",
       "id": "id1",
       "log_type": "aws_cloud_watch",
+      "source": "accountId-region",
```

Comment:
Minor: 'accountId-region' -> should this be 'accountId-Region' as we are using 'R' in region?